### ISSUE TITLE
Enhance presentation mode 

### ISSUE THREAD

**Issue by blobinabottle** [ludushq/issues#15]
This will be part of a next big step.
- Speaker notes.
- Timing
- Commenting / reaction
- More secret features

Related people:
https://www.facebook.com/groups/ludusPrivate/permalink/1752461735009147/

**Comment by jlvan** [comment]
+ I think it would be easier to, at the end of the video, enable the arrow navigation again

**Comment by jlvan** [comment]
Be able to present a ludus presentation on a second monitor but open it from the main monitor.
Context : when having two monitors set up. The second one has the color black as a «wallpaper» so to say, so when the user doesn't have the presentation in presentation mode, the monitor (or beamer) only shows black. and when the user opens the presentation in presentation mode, it shows on the second monitor and on the main monitor a have the moderator view

**Comment by vinch** [comment]
I don't think this is possible in a browser @jlvan. But we have solutions like Henry Lim's (or the one we'll build at some point) to be able to control the presentations from another screen.

**Comment by blobinabottle** [comment]
I do it everyday.... No sure to follow there.
